sys/valexpr/src/ConstDefs.hs

```diff
@@ -31,3 +32,12 @@ data Const = Cbool    { cBool :: Bool }
   deriving (Eq, Ord, Read, Show, Generic, NFData, Data)
 
 instance Resettable Const
+
+instance SortOf Const where
+  sortOf (Cbool _b)                        = sortId_Bool
```

Comment:
Why not using plain wild cards?

Reply:
just a copy...
Probably 
1. was b in the past.
 2. hlint complaints b isnt used
3. add _ to silence hlint

Reply:
When one has multiple unused arguments, and want to use one names help in picking the right one.